secant/Features/SendFlow/SendFlowStore.swift

```diff
@@ -23,6 +23,7 @@ struct SendFlowState: Equatable {
         case done
     }
 
+    var addMemoCheckCircleState: CheckCircleState
```

Comment:
I personally would omit the CheckCircle from the variable's name. 
```swift
var addMemoState: CheckCircleState
```
similarly like 2 line bellow
```swift
var memoState: MultiLineTextFieldState
```
We don't call it memoMultiLineTextFieldState: MultiLineTextFieldState. Typically it's noticeable when pulling back
```swift
       state: \SendFlowState.addMemoCheckCircleState,
        action: /SendFlowAction.addMemo,
 ```
The inconsistency in naming between Action and State.